### ISSUE TITLE
[feature request] Title field within contact information

### ISSUE THREAD

**Issue by massey21** [usnistgov/NEMO#195]
Request:
When entering contact information for people in our facility, it would be helpful to be able to enter descriptive titles/roles for each person -- just an additional one-line text field.  

Purpose:
I want to be able to label this person as the operations manager, that person as the safety officer, this other person as the equipment support supervisor, the next person as chief engineer, someone else as our administrative support, etc.

**Comment by rptmat57** [comment]
It might not be as flexible as a free text field, but you can set a list of possible User Types in Detailed administration -> NEMO -> User Types and set those on people

**Comment by massey21** [comment]
That user type information doesn't show up on the contact info page, though!  Also, not all names in the contact list are facility users -- our ES&H team is a perfect example.  Here I've assigned Anand the user type "operations manager" and Manish the user type "safety officer," and I gave them skeleton contact info where the contact is linked to their NEMO username.  
![image](https://github.com/usnistgov/NEMO/assets/98626874/df39516d-6165-4401-bbbb-3dc5ec7ae482)

This might be a better example of the per-person type of role info I need to capture.
![image](https://github.com/usnistgov/NEMO/assets/98626874/5fd5cdc6-7899-426f-814c-c78aa5b1e8e0)
